### PR TITLE
⚡  Update templates to Expo 51, Next.js 14.2, Turbo 2; migrate to bun

### PR DESCRIPTION
**Pull request by danger9224**
This PR upgrades the example projects to turbo 2.0.9 (with some change in turbo.json); Expo 51.0.23 (from 49, 2 main versions behind); Next.js 14.2.5 (from 13.5); and migrated from yarn to bun, as this has cut deployment time in half on an empty project:

image

### CODE REVIEW

**Comment by nandorojo** [comment]
don’t have time to test so i’m trusting that you tested all platforms!

File: package.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
     "image/**/*",
     "navigation/**/*"
   ],
-  "packageManager": "yarn@1.22.19",
+  "packageManager": "yarn@4.3.1",
```
Comment (nandorojo) [inline package.json:62]: shouldn’t this be bun?

Comment (jeffreyvalle) [reply package.json:62]: "packageManager": "bun@1.1.18" is breaking yarn install, changing to bun to yarn@3.4.1 or a version that exists/is downloadable works. Previously the yarn version was included in the monorepo, if you have yarn@1.1.18 to include in the project as before, that may work. I don't know anything, I'm just guessing. Either way, this broke things.

**Comment by nandorojo** [comment]
@dnlgrgly could you please see #487? This seems to have broken the examples for people. Could we go back to yarn? Bun is less widely adopted and seems to have issues.

Thank you!

**Comment by dnlgrgly** [comment]
@nandorojo I think it's best to revert this PR given the number of issues it caused. Sorry about the hiccups. 😭 

Due to some vacations I won't have enough time to fix this in a timely manner, but I'll give it another try in the following weeks. 🙏

**Comment by nandorojo** [comment]
Any shot you could submit a PR that reverts it?

Thank you!

**Comment by nandorojo** [comment]
Hey @dnlgrgly just following up on reverting this if possible, getting many open issues about it…also open to just fixing the issues people are getting with the tailwind template. Thank you!

**Comment by dnlgrgly** [comment]
@nandorojo I've opened a PR that reverts this one here: https://github.com/nandorojo/solito/pull/492

**Pull request merged**